diktat-rules/src/test/resources/test/smoke/build.gradle_.kts

```diff
@@ -11,14 +11,14 @@ repositories {
     maven { url = uri("https://example.com") }
 }
 
-tasks.register("generateAvailableRules") {
+val generateAvailableRules = tasks.register("generateAvailableRules") {
```

Comment:
Also, will `val gegenerateAvailableRules by tasks.registering { ... }` be allowed?

Reply:
Yes, changed it in test